paymentsheet/api/paymentsheet.api

```diff
@@ -1249,6 +1249,30 @@ public final class com/stripe/android/paymentsheet/paymentdatacollection/polling
 	public synthetic fun newArray (I)[Ljava/lang/Object;
 }
 
+public final class com/stripe/android/paymentsheet/state/GooglePayState$Available$Creator : android/os/Parcelable$Creator {
```

Comment:
Looks like we need to add some `@RestricTo` calls on these classes. Feels like we shouldn't need it, but it'll remove these from the diff.